test/lib/next-modes/base.ts

```diff
@@ -613,4 +613,11 @@ export class NextInstance {
       cb(...args)
     })
   }
+
+  public getCliOutputFromHere() {
```

Comment:
nice, this is a fairly common pattern  👍

Reply:
nice 👍